Documentation/ApiOverview/Logging/Logger/Index.rst

```diff
@@ -31,6 +31,37 @@ Constructor injection can be used to automatically instantiate the logger:
    }
 
 
+It is also possible to use the :php:`LoggerAwareInterface`:
+
+.. code-block:: php
+
+   namespace Psr\Log;
+
+   /**
+    * Basic Implementation of LoggerAwareInterface.
+    */
+   trait LoggerAwareTrait
```

Comment:
It doesn't make sense to embed the trait source here, especially not when the sentence before talks about the `LoggerAwareInterface`.

Instead there should be another `MyClass` example like above using this interface and trait.